src/renderer/components/ProgramChangeView.tsx

```diff
@@ -88,12 +98,23 @@ function ProgramChangeView(props: {
   const [programID, setProgramID] = useState(KnownProgramID.SystemProgram);
 
   useEffect(() => {
-    subscribeProgramChanges(net, programID, setChangesState);
+    function sortFunction(a: AccountInfo, b: AccountInfo) {
+      switch (sortColumn) {
+        case 4: // count
+          return b.count - a.count;
```

Comment:
this one could do with a little fuzz, when the top 3 accounts are _almost_ in lockstep, the batching can make them switch places in the list constantly.